release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -583,6 +583,11 @@ The Cluster Samples Operator is deprecated with the {product-title} 4.16 release
 
 The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated for several versions. However, in this release, the CRD is completely removed. The sole supported method of configuring MetalLB address pools is by using the `IPAddressPools` CRD.
 
+[id="ocp-4-16-service-binding-operator-documentation-removed_{context}"]
+==== Service Binding Operator documentation removed
+
+With this release, the documentation for the Service Binding Operator has been removed. The Service Binding Operator includes a controller and a custom resource definition (CRD) for service binding.
```

Comment:
I'd recommend putting in a brief explanation on why it's being removed (because the operator is EOL/no longer supported/available?)

Also, if it's being removed, I'd argue that the second sentence explaining what it was used for isn't necessary.

Reply:
Hi @bergerhoffer - I was told that the SBO operator was deprecated a few releases ago and they want the content removed from this release. I am unsure if support for this operator has been completely removed. WDYT?

Reply:
Hello @bergerhoffer - I just had a chat with the concerned SME and they stated that we can use either of the first two reasons to answer why this documentation was removed. So, I suggest the following RN entry re-write. Do you think that is correct:

With this release, the documentation for the Service Binding Operator (SBO) has been removed as support is no longer provided for this Operator.

Reply:
Great thanks for checking on that! I'd change "as" to "because" (per IBM Style guide guidance) but otherwise that sounds fine to me. Or "because this Operator is no longer supported". Whatever you prefer

Reply:
Yeah, changing it to "With this release, the documentation for the Service Binding Operator (SBO) has been removed because this Operator is no longer supported."